docs/source/usersguide/depletion.rst

```diff
@@ -177,3 +192,163 @@ across all material instances.
+:class:`~openmc.deplete.IndependentOperator` using the
+:class:`~openmc.deplete.MicroXS` class::
+
+    import openmc
+
+    model = openmc.Model.from_xml()
+
+    micro_xs = openmc.deplete.MicroXS.from_model(model, model.materials[0])
+
+The :meth:`~openmc.deplete.MicroXS.from_model()` method will produce a
+:class:`~openmc.deplete.MicroXS` object with microscopic cross section data in
+units of ``b``, which is what :class:`~openmc.deplete.IndependentOperator`
+expects the units to be. The :class:`~openmc.deplete.MicroXS` class also includes functions to read in cross section data directly from a ``.csv`` file or from data arrays::
+
+    micro_xs = MicroXS.from_csv(micro_xs_path)
+
+    nuclides = ['U234', 'U235', 'U238']
+    reactions = ['fission', '(n,gamma)']
+    data = np.array([[0.1, 0.2],
+                     [0.3, 0.4],
+                     [0.01, 0.5]])
+    micro_xs = MicroXS.from_array(nuclides, reactions, data)
+
+.. important ::
+
+   Both :meth:`~openmc.deplete.MicroXS.from_csv()` and
+   :meth:`~openmc.deplete.MicroXS.from_array()` assume the cross section values 
+   provided are in barns by defualt, but have no way of verifying this. Make
+   sure your cross sections are in the correct units before passing to a
+   :class:`~openmc.deplete.IndependentOperator` object.
+
+Caveats
+-------
+
+Reaction Rate Normalization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`~openmc.deplete.IndependentOperator` class supports two methods for
+normalizing reaction rates:
+
+.. important::
+
+   Make sure you set the correct parameter in the :class:`openmc.abc.Integrator`
+   class. Use the ``source_rates`` parameter when
+   ``normalization_mode == source-rate``, and use ``power`` or ``power_density``
+   when ``normalization_mode == fission-q``.
+
+1. ``soure-rate`` normalization, which assumes the ``source-rate`` provided by
+   the time integrator is a flux, and obtains the reaction rates by multiplying
+   the cross-sections by the ``source-rate``.
+2. ``fission-q`` normalization, which assumes the ``source-rate`` provided by
+   the time integrator is a power, and obtains the reaction rates by computing a
+   value for the flux based on this power. The general equation for the flux is 
+
+   .. math::
+
+      \phi = \frac{P}{V \cdot \sum_i (Q_i \cdot \Sigma^f_i \cdot \rho_i)}
+
+   where :math:`\sum_i` is the sum over all nuclides :math:`i`. This equation
+   makes the same assumptions and issues as discussed in
+   :ref:`energy-deposition`. Unfortunately, the proposed solution in that
+   section does not apply here since we are decoupled from transport code.
+   However, there is a method to converge to a more accurate value for flux by
+   using substeps during time integration.
+   `This paper <https://doi.org/10.1016/j.anucene.2016.05.031>`_ provides a
+   good discussion of this method. Hopefully such a method will be implemented
+   in OpenMC in the near future.
```

Comment:
I try to avoid making promises in the documentation
```suggestion
   good discussion of this method.
```

Reply:
haha that's a good recommendation